keyboards/sneakbox/disarray/ortho/disarray.c

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright 2020 Bryan Ong
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include "disarray.h"
+
+void keyboard_pre_init_kb() {
+    setPinOutput(B1);
+    keyboard_pre_init_user();
+}
+
+bool led_update_kb(led_t led_state) {
+    bool res = led_update_user(led_state);
+    if(res) {
+        writePin(B1, led_state.num_lock);
+    }
+    return res;
+}
```

Comment:
This can all be replaced with `#define LED_NUM_LOCK_PIN B1` in your config.h file

Reply:
thank you. I have made the change